shared/templates/kernel_module_disabled/tests/file_not_there.fail.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+rm -f /etc/modprobe.d/{{{ KERNMODULE }}}.conf
```

Comment:
This test should probably go through all of the directories checked, not just `/etc/modprobe.d`.

And looking at the check, it looks for any `*.conf` file with `install {{{ KERNMODULE }}}`.
So the test should remove any `*.conf` files in any of the directories checked with `install {{{ KERNMODULE }}}`.

Reply:
The `kernel_module_preparation.sh` has some code that can be reused.